docs/content/about-iota/iota-wallet/how-to/import/balance-finder.mdx

```diff
@@ -0,0 +1,46 @@
+---
+description: A guide on how to find and use the balance finder in the IOTA Wallet.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet]
+pagination_prev: about-iota/iota-wallet/how-to/import
+---
+
+# Use the Balance Finder
```

Comment:
I would maybe try to clarify to users that they could have off balances here, basically if they see the labels "vesting" and or "migration" it means they may require a manual action to check vesting schedules o migrate from stardust so they balance becomes accurate

Reply:
Yes, the thing is that "If you still have any missing balances, hit `Search` again." is not really accurate, search again is about finding missing addresses, but the found addresses can have missing balances because they require migration or have a vesting schedule

Reply:
Updated in [1afc1dc](https://github.com/iotaledger/iota/pull/6979/commits/1afc1dc6b639b92bcdd92a10d1c9221b2410c79a)